gapis/api/gvr/gvr.go

```diff
@@ -39,6 +39,8 @@ func (s *State) Root(ctx context.Context, p *path.State) (path.Node, error) {
 
 func (*State) SetupInitialState(ctx context.Context) {}
 
+func (s *State) InitializeCustomState() {}
```

Comment:
Do we need this and `SetupInitialState`?

Reply:
Unfortunately yes. We need SetupInitialState to take an initial state block, and make sure that it is consistent. 

InitializeCustomState takes an empty state block and sets up all of the objects within.